### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by kkujawa-sumo**
- add instruction for installation form Red Hat Marketplace
- refactor of documentation
  - add section to main README.md (Verification, Viewing Data in Sumo Logic)
  - replace section for installation from OperatorHub with section for installation from RedHatMarketplace

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,18 +57,50 @@ Sumo Logic Kubernetes Collection Helm Operator for the [Sumo Logic Kubernetes Co
...
-section for Sumo Logic Kubernetes Collection Helm Chart.
-
-Example configurations for Sumo Logic Kubernetes Collection Helm Operator are available in [config/samples](config/samples) directory.
+### Verification
+
+Verify that Pod related to Sumologic Kubernetes Collection are running,
+for default configuration you should see similar list of Pods:
+
+```bash
+$ kubectl get pods -n openshift-operators
```
Comment (perk-sumo) [inline README.md:66]: Is it really in the `openshift-operators` namespace?

**Review by perk-sumo — APPROVED** [review]
LGTM 👍

File: docs/install_from_redhat_marketplace.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+1. For further information, see the [Red Hat Marketplace Operator documentation][red-hat-marketplace-operator-docs].
+
+[red-hat-marketplace-docs]: https://marketplace.redhat.com/en-us/documentation/clusters
+[red-hat-marketplace-operator-docs]: https://marketplace.redhat.com/en-us/documentation/operators
+
+![redhat_marketplace_installation](../images/redhat_marketplace_installation.png)
+
+## Verification of operator installation
+
+1. Once status changes to Up to date, click the vertical ellipses and select Cluster Console.
```
Comment (astencel-sumo) [inline docs/install_from_redhat_marketplace.md:44]:
```suggestion
1. Once status changes to Up to date, click the vertical ellipsis `⋮` and select Cluster Console.
```

**Review by astencel-sumo — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (kkujawa-sumo) [reply README.md:66]: Yes, this namespace is used to set operator scope to the whole cluster (and this is selected by default). It is possible to deploy operator in different namespace but then scope of this operator is limited only to this namespace.

**Pull request merged**